app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/MainFragment.java

```diff
@@ -87,7 +87,7 @@ private ListRow createCardRow(CardRow cardRow) {
 
     private void setupUIElements() {
         setTitle(getString(R.string.browse_title));
-        setBadgeDrawable(getResources().getDrawable(R.drawable.title_android_tv, null));
+        setBadgeDrawable(getResources().getDrawable(R.drawable.title_android_tv));
```

Comment:
Resources.getDrawable(int) is deprecated. You should prefer Context.getDrawable() or passing a valid theme to the 2-arg version.